docs/user-guide/persistent-volumes/index.md

```diff
@@ -206,6 +220,36 @@ Claims can specify a [label selector](/docs/user-guide/labels/#label-selectors)
 
 All of the requirements, from both `matchLabels` and `matchExpressions` are ANDed together – they must all be satisfied in order to match.
 
+### Class
+
+A claim can request a particular class by specifying the name of a
+`StorageClass`using the annotation `volume.beta.kubernetes.io/storage-class`.
+Only PVs of the requested class, ones with the same annotation as the PVC, can
+be bound to the PVC.
+
+PVCs don't necessarily have to request a class, they can omit the annotation or
+set it to `""`. The cluster treats PVCs that don't request a class differently
```

Comment:
It's slightly more complicated.

* Missing class annotation in PVC means "I don't care about the class" and defaulting in the admission controller happens, i.e. the user gets the default class.

  * If there is no default storage class set or the admission controller is not running, a classless PV is then requested (see below).

* Class annotation set to "" means that the PVC requests PV with class="" or missing annotation, i.e. a classless PV. That's the only way to get a PV with no/empty class.